tests/Unit/Model/ResetPasswordRequestTraitTest.php

```diff
@@ -26,21 +28,28 @@ public function testIsCompatibleWithInterface(): void
 
     public function propertyDataProvider(): \Generator
     {
-        yield ['selector', '@ORM\Column(type="string", length=20)'];
```

Comment:
these unit tests almost seem silly if we have a full stack test in the bundle.. if we dont - we should create one and remove these tests..